src/main/java/visualizations/BarChart.java

```diff
@@ -0,0 +1,37 @@
+package visualizations;
+
+
+import tech.tablesaw.api.Table;
+import tech.tablesaw.plotly.Plot;
+import tech.tablesaw.plotly.api.HorizontalBarPlot;
+import tech.tablesaw.plotly.api.VerticalBarPlot;
+
+/**
+ * Class that plots basic Bar charts.
+ */
+public class BarChart {
```

Comment:
Perhaps ```BarPlot``` is a better name????

Reply:
I think I'll revert the naming to what it was. it's clear enough and it doesn't conflict with tablesaw. what do you say.

Reply:
Ok do it as you said. I see your point